### ISSUE TITLE
Simple Todo App - Review my app please. Thank you!

### ISSUE THREAD

**Issue by minhtien1811** [minhtien1811/Submitting-Simple-Todo-App-Prework#2]
Here is my completed Simple Todo App Prework! Thank you.

/cc @coderschoolreview

**Comment by coderschoolreview** [comment]
### Thank you for your Android submission, minhtien1811!

 :laughing: Everything looks good with the format of your submission. We'll be reviewing your submission soon!

**Comment by minhtien1811** [comment]
Thank you very much!

**Comment by hoanle** [comment]
Hi  @minhtien1811 
Could u please reupload the whole prj. There are several files missing

**Comment by minhtien1811** [comment]
Yes, i'm at school right now, but when i go home i will do it! Thank you
for email me :)

**Comment by minhtien1811** [comment]
@hoanle
Hello!, i just got home, I reuploaded my whole project on new branch call "reupload", u can go and check that out :smile:

**Comment by hoanle** [comment]
Hi @minhtien1811 

Thanks for the submission

In general
- The app finished all the required tasks. No optional task.
- A nice layout. I really like your About Me page
- Don't understand what the setting page is for? Just pick a colour and that's it?
- You could re-use code at many places (Enter new Item & Edit Item)
- One of your layout (about me) is unnecessary deep (6 levels). It can be constructed with much simpler layout
- You could try using SQLite database, DialogFragment, writing your own adapter to make the app better

One more thing, the deadline of submission is 1st March. It means that you are a bit late.

**Comment by minhtien1811** [comment]
Thanks @hoanle
- The setting page is for changing App's color theme, but like i said in
  Readme, i don't have enough time for that :(
- i have submitted my project on Feb 28, but i submit it in wrong way @@
- I received an email "CoderSchool Android Pre-work -- Checking in" on
  March 2, it said the deadline is 1st March, in that moment i realized the
  submission on Feb 28 was wrong (because I never use GitHub, don't know how
  it's work until now :( ). And on March 2, i know it's late but i want to
  join the Android class so bad that why i take another shot, google Github
  and submit my project again.
